scripts/run-pre-build-checks

```diff
@@ -13,7 +13,7 @@ bash ${x_option:+-x} ./scripts/enforce-change-notes-updated-on-pr-to-master
 ./scripts/enforce-newline-at-eof
 ./scripts/enforce-settings-gradle-match-directories
 ./scripts/enforce-to-string-call-super-in-subclasses
-./scripts/enforce-unique-key-shortcuts-for-action-texts
+############## ./scripts/enforce-unique-key-shortcuts-for-action-texts
```

Comment:
@mkondratek since #407 most shortcuts are repeated, but maybe it's okay... TBD what should be done with that

Reply:
(and this script was not failing since the path to bundle was no longer correct)

Reply:
yeah, I think we can live without this script

Reply:
Removed